app/Http/Resources/PackageDetailResource.php

```diff
@@ -22,6 +22,16 @@ public function toArray($package)
 
             if (! is_null($packagistData)) {
                 $composer_latest = $this->extractStableVersionsFromPackages($packagistData)->first();
+
+                if (! is_null($composer_latest)) {
+                    $composer_requirements = $composer_latest['require'];
```

Comment:
Also check `require-dev`? 

Code could also be simplified to null coalesce, this is personal preference though:
```php
$novaVersion = $composer_latest['require']['laravel/nova']
    ?? $composer_latest['require-dev']['laravel/nova']
    ?? null
```

Reply:
Agreed I'll make the changes shortly

Reply:
> Agreed I'll make the changes shortly.
> 
> In regards to the require-dev I believe we came to the agreement not to show nova 4 support unless it was specifically in require.

Reply:
Its up to you guys, but I have seen some release 4.x support by only including it in require-dev, not sure of the reasoning myself.

Reply:
I think it will be best if we stick to only `require` for now and if it becomes an issue where `require-dev` is needed then we could add that in